File: app/src/main/java/com/siyeon/haniumproject/CCTVActivity.java
package com.siyeon.haniumproject;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.view.View;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class CCTVActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.cctv_layout);

        @SuppressLint("WrongViewCast") Button returnButton = findViewById(R.id.button_return);
        returnButton.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View v) {
                new MainActivity();
            }
        });
    }

    @Override
    protected void onStart() {
        super.onStart();

        WebView webView = (WebView) findViewById(R.id.webView);
        webView.setWebViewClient(new WebViewClient());
        webView.setBackgroundColor(125);


        WebSettings webSettings = webView.getSettings();
        webSettings.setJavaScriptEnabled(true);


        webView.loadDataWithBaseURL("https://aries574.tistory.com/81", "<html><head></head><body>" +"<img src='/stream/video.mjpeg' alt='image'></body></html>",
                "text/html", "UTF-8", "");

    }
}

File: app/src/main/java/com/siyeon/haniumproject/CctvActivity.kt
package com.siyeon.haniumproject

import android.os.Bundle
import android.webkit.WebChromeClient
import android.webkit.WebView
import android.webkit.WebViewClient
import android.widget.ImageButton
import androidx.appcompat.app.AppCompatActivity

class CctvActivity: AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.cctv_layout)

        // java activity 파일 삭제하고 입력한 코드
        val webView = findViewById<WebView>(R.id.webView)
        webView.apply {
            webViewClient = WebViewClient()
            webChromeClient = WebChromeClient()
            settings.javaScriptEnabled = true
        }

        webView.loadUrl("https://www.youtube.com/watch?v=6Dbq05hchX4")
        //여기까지


        title = "CCTV 확인"

        var buttonReturn = findViewById<ImageButton>(R.id.button_return)
        buttonReturn.setOnClickListener {
            finish()
        }
    }



}